platform/src/default.rs

```diff
@@ -53,6 +59,20 @@ impl Platform for DefaultPlatform {
         Ok(issuer_name.len())
     }
 
+    fn get_issuer_sn(&mut self, out: &mut [u8; MAX_SN_SIZE]) -> Result<usize, PlatformError> {
+        let sn = X509::from_pem(TEST_CERT_PEM)
```

Comment:
nit: maybe we should make a const value for the X509 representation of TEST_CERT_PEM so we don't have to repeat work in get_issuer_name and get_issuer_sn when converting the PEM to an X509?